docs/README.md

```diff
@@ -1,154 +1,233 @@
 # Docker Documentation
 
-The source for Docker documentation is here under `sources/` and uses extended
-Markdown, as implemented by [MkDocs](http://mkdocs.org).
+The source for Docker documentation is in this directory under `sources/`. Our
+documentation uses extended Markdown, as implemented by
+[MkDocs](http://mkdocs.org).
 
-The HTML files are built and hosted on
-[http://docs.docker.com](http://docs.docker.com), and update automatically
-after each change to the `docs` branch of [Docker on
-GitHub](https://github.com/docker/docker) thanks to post-commit hooks.
+Through the use of post-commit hooks, Docker builds and updates our static HTML documentation automatically after each change to the `docs` branch of [this repository](https://github.com/docker/docker). You can find the resulting documentaiton hosted on [http://docs.docker.com](http://docs.docker.com).
 
-## Contributing
+## Understanding the documentation branches and processes
 
-Be sure to follow the [contribution guidelines](../CONTRIBUTING.md).
-In particular, [remember to sign your work!](../CONTRIBUTING.md#sign-your-work)
+Docker has two primary branches for documentation:
 
-## Getting Started
+| Branch   | Description                    | URL (published via commit-hook)                                              |
+|----------|--------------------------------|------------------------------------------------------------------------------|
+| `docs`   | Official release documentation | [http://docs.docker.com](http://docs.docker.com)                             |
+| `master` | Merged but unreleased development work    | [http://docs.master.dockerproject.com](http://docs.master.dockerproject.com) |
 
-Docker documentation builds are done in a Docker container, which installs all
-the required tools, adds the local `docs/` directory and builds the HTML docs.
-It then starts a HTTP server on port 8000 so that you can connect and see your
-changes.
+Additions and updates to upcoming releases are made in a feature branch off of the `master` branch. The Docker maintainers also support a `docs` that contains the last release of documentation. 
 
-In the root of the `docker` source directory:
+After a release, documentation updates are continually merged into `master` as they occur. This work includes new documentation for forthcoming features, bug fixes, and other updates. 
 
-    $ make docs
-    .... (lots of output) ....
-    docker run --rm -it  -e AWS_S3_BUCKET -p 8000:8000 "docker-docs:master" mkdocs serve
-    Running at: http://0.0.0.0:8000/
-    Live reload enabled.
-    Hold ctrl+c to quit.
+Periodically, the Docker maintainers update `docs.docker.com` between official releases of the Docker product. They do this by cherry-picking commits from `master`, merging them into `docs`,  and the publishing the result.
```

Comment:
..."the publishing..." --> "...then publishing...

Also, not sure "product" is the right word here, since this is open source. Maybe "project"? or just "Docker"?